CHANGELOG.rst

```diff
@@ -11,6 +11,11 @@ Latest
 * Support warnings in contract checks.
 * Add unmatched_ignore_imports_alerting option for each contract.
 
+1.2.7 (2022-03-24)
```

Comment:
This should just go in the `latest` section at the top, as we don't know what release it will appear in yet.

Reply:
:doh: fixing now :)